meineLoesungen/functional.js

```diff
@@ -0,0 +1,55 @@
+let zahlen = [5,6,7,8]
+
+// 1
+{
+    // Funktional
+    function sum(array){
+        // Startwert || vorangegangener Wert:sum
+        // Aktuelle Zahl im Array:val
+        return array.reduce((sum,val) =>sum + val, 0);
+    }
+    console.log(sum(zahlen))
+
+    // Imperativ
+    let summe = zahlen.reduce((sum, val) =>sum + val)
```

Comment:
mit imperativ ist gemeint: keine funktionalen konzepte verwenden. sprich: mit for schleife arbeiten.